packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -109,10 +109,11 @@ export default (new Bundler({
             let bundle = bundleGraph.createBundle({
               entryAsset: asset,
               isEntry:
+                dependency.bundleBehavior === 'inline' ||
                 asset.bundleBehavior === 'inline'
                   ? false
                   : dependency.isEntry || dependency.needsStableName,
-              isInline: asset.bundleBehavior === 'inline',
+              bundleBehavior: dependency.bundleBehavior ?? asset.bundleBehavior,
```

Comment:
Which one should win? 😉

Reply:
I would say keep it `bundleBehavior`, it's easier to search since it's all the same feature

Reply:
I think @devongovett is asking if the dependency's behavior or asset's behavior should be preferred

Reply:
yeah I think the dep should win because the importing code might have an expectation about what the asset exposes.